cluster-scope/base/core/namespaces/vault/resourcequota.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: vault-custom
+spec:
+  hard:
+    requests.cpu: '500'
```

Comment:
500 cores?!? did you mean `500m` as in millicores?

Reply:
Yikes yeah 500 cores is probably overkill... updating to millicores